app/views/admin/overview/index.html.erb

```diff
@@ -8,17 +8,17 @@
         <h3 class="crayons-subtitle-1">Welcome Checklist</h3>
         <p class="crayons-subtitle-4 color-base-70 mb-3">Here are a few things we recommend doing now that you're using Forem!</p>
         <ol>
-          <li class="mb-1 -ml-4">Set up and configure your Forem under <%= link_to "Config", admin_config_path, 'data-action': "click->ahoy#trackOverviewLink" %></li>
-          <li class="mb-1 -ml-4"><%= link_to "Invite people", admin_invitations_path, 'data-action': "click->ahoy#trackOverviewLink" %> to join your Forem</li>
-          <li class="mb-1 -ml-4"><%= link_to "Update your Forem tags", admin_tags_path, 'data-action': "click->ahoy#trackOverviewLink" %></li>
-          <li class="mb-1 -ml-4"><%= link_to "Create pages", admin_pages_path, 'data-action': "click->ahoy#trackOverviewLink" %> and <%= link_to "sort them in the navigation sidebar", admin_navigation_links_path, 'data-action': "click->ahoy#trackOverviewLink" %></li>
-          <li class="mb-1 -ml-4">Read the <a href="https://forem.gitbook.io/forem-admin-guide/quick-start-guide" data-action="click->ahoy#trackOverviewLink">Quick Start Guide</a></li>
-          <li class="mb-1 -ml-4"><%= link_to "Set up a Welcome Thread", admin_welcome_index_path, 'data-action': "click->ahoy#trackOverviewLink" %></li>
-          <li class="mb-1 -ml-4">Browse the <a href="https://forem.gitbook.io/forem-admin-guide" data-action="click->ahoy#trackOverviewLink">Forem Admin Guide</a></li>
+          <li class="mb-1 -ml-4">Set up and configure your Forem under <%= link_to "Config", admin_config_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4"><%= link_to "Invite people", admin_invitations_path, "data-action": "click->ahoy#trackOverviewLink" %> to join your Forem</li>
+          <li class="mb-1 -ml-4"><%= link_to "Update your Forem tags", admin_tags_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4"><%= link_to "Create pages", admin_pages_path, "data-action": "click->ahoy#trackOverviewLink" %> and <%= link_to "sort them in the navigation sidebar", admin_navigation_links_path, "data-action": "click->ahoy#trackOverviewLink" %></li>
+          <li class="mb-1 -ml-4">Read the <a href="https://admin.forem.com/quick-start-guide" data-action="click->ahoy#trackOverviewLink">Quick Start Guide</a></li>
```

Comment:
I think this is /docs/quick-start-guide (the last changed line in the PR was good).